src/inputs/RichTextField.tsx

```diff
@@ -130,22 +119,63 @@ export function RichTextField(props: RichTextFieldProps) {
     }
   }, [value, readOnly]);
 
+  useEffect(() => {
+    if (parentRef.current) {
+      function onEditorInit() {
+        editor.current = (editorElement.current as any).editor;
+        if (mergeTags !== undefined) {
+          attachTributeJs(mergeTags, editorElement.current!);
+        }
+
+        currentHtml.current = value;
+        editor.current!.loadHTML(value || "");
+      }
+
+      parentRef.current.addEventListener("trix-initialize", onEditorInit);
+      setShowTrix(true);
+
+      return () => {
+        if (parentRef.current) {
+          parentRef.current.removeEventListener("trix-initialize", onEditorInit);
+        }
+      };
+    }
+  }, [parentRef]);
```

Comment:
Does this need to be `parentRef.current`? I don't think `parentRef` itself will ever change, unless that is your intent? If so, maybe just use `[]`? ...I'm ~pretty sure that refs are ignored in the usual "check that variables you access in the lambda are in the deps list" rule.